cmd/gobrew/main.go

```diff
@@ -30,18 +31,17 @@ var allowedArgs = []string{
 }
 
 func init() {
-	log.SetFlags(0)
 
 	if !isArgAllowed() {
-		log.Println("[Info] Invalid usage")
-		log.Print(usage())
+		utils.Infof("[Info] Invalid usage")
+		fmt.Println(usage())
```

Comment:
If we use utils.Infof, why we cant to use it here?

Reply:
Let me check

Reply:
Not using Infof, we don't want to add extra `==>` as a prefix to the output. It is just an indication that it is raw output. It is better to keep it this way.

Reply:
We can use Infoln
https://github.com/kevincobain2000/gobrew/blob/master/utils/utils.go#L92

without extra symbols